pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -394,7 +394,11 @@ func (o *LoginOptions) SaveConfig() (bool, error) {
 	}
 
 	if err := kclientcmd.ModifyConfig(o.PathOptions, *configToWrite, true); err != nil {
```

Comment:
Hm, what if I type the errors in [loader.go#WriteToFile](https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/client/unversioned/clientcmd/loader.go#L376) and then return a message accordingly in loginoptions.go?